node.gyp

```diff
@@ -1476,7 +1476,7 @@
     }], # end aix section
     # TODO(RaisinTen): Enable this to build on other platforms as well.
     ['(OS=="mac" or (OS=="linux" and target_arch=="x64")) and \
-      node_use_openssl=="true"', {
+      node_use_openssl=="true" and node_shared_openssl=="false"', {
```

Comment:
Tests are failing. I think you need to add this conditional:
```js
if (process.config.variables.node_shared_openssl === false)
```
around https://github.com/nodejs/node/blob/eadfbaeba2b65377951219a83775a76b590f85e3/test/parallel/test-crypto-engine.js#L31 to fix the failure.